mode/fortran/fortran.js

```diff
@@ -112,7 +116,7 @@ CodeMirror.defineMode("fortran", function() {
                      "c_short", "c_signed_char", "c_size_t", "character",
                      "complex", "double", "integer", "logical", "real"]);
   var isOperatorChar = /[+\-*&=<>\/\:]/;
-  var litOperator = new RegExp("(\.and\.|\.or\.|\.eq\.|\.lt\.|\.le\.|\.gt\.|\.ge\.|\.ne\.|\.not\.|\.eqv\.|\.neqv\.)", "i");
```

Comment:
Any idea what these (uselessly) escaped period characters might have been for? I don't know Fortran, but if I google them I see these operators surrounded by actual periods (which should have been `\\.` in this string).

Also the new code drops the case-insensitive flag. Is that intentional?